app/assets/stylesheets/assignments.scss

```diff
@@ -206,10 +206,8 @@ span.info_date {
   margin-right: 5px;
 }
 
-#invite_dialog {
-  fieldset {
-    padding-top: 1.5em;
-  }
+#invite_dialog fieldset {
```

Comment:
Avoid using id selectors<br>Selector `invite_dialog` should be written in lowercase with hyphens